aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,106 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get Started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you will create a SignalR on ASP.NET Core application.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core.
+
+  ![Solution](get-started-signalr-core/_static/signalr-get-started-finished.png)
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Create a SignalR on ASP.NET Core web app.
```

Comment:
SignalR  what? client?

Reply:
I think this should say something like "Create an ASP.NET Core SignalR app".

Reply:
"Create a SignalR on ASP.NET Core web app" is the thing development task they will do.   
Do you mean to add something like "containing server and client libraries"? The question isn't clear.
For consistency - None of the other tutorials describe what's in a proj template in the "development tasks" checklist.  The steps are supposed to say what you'll accomplish, not describe what is in the product.  What is in the proj template is covered during the tutorial anyway.

Reply:
>Create a SignalR on ASP.NET Core web app.

Doesn't make any sense. A SignalR what?
The template is secondary to the tutorial.  

> "containing server and client libraries

Yes, or to host the SignalR server and client. It's more than the client libs, you supply the client app.

Reply:
>Create an ASP.NET Core SignalR app

But there's really no such thing. We're creating a client and server app  together.  We should try to make that clear - client and server sides of SignalR.

Reply:
How about just: "Create an ASP.NET Core web app with a SignalR Hub".